rosette/h/RblAtom.h

```diff
@@ -23,6 +23,8 @@
 #include "rosette.h"
 #include "Ob.h"
 
+#define protected public
```

Comment:
Why not just replace the `protected:` directives with `public:`?

Reply:
Done.